python/samples/demos/guided_conversations/pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[tool.poetry]
+name = "guided_conversation"
+version = "0.1.0"
+description = ""
+authors = ["DavidKoleczek <dkoleczek@microsoft.com>", "natalieisak", "christyang-ms", "dasham8"]
+license = "MIT"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10,<3.13"
+
+semantic-kernel = { version = "==1.1.1"}
```

Comment:
Do we need to pin to a specific SK version?

Reply:
Nope, switched to using caret requirements for all dependencies.